common/src/main/java/com/wynntils/models/gearinfo/type/GearInstance.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.gearinfo.type;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.models.concepts.Powder;
+import com.wynntils.models.stats.StatCalculator;
+import com.wynntils.models.stats.type.StatActualValue;
+import com.wynntils.models.stats.type.StatPossibleValues;
+import com.wynntils.models.stats.type.StatType;
+import java.util.DoubleSummaryStatistics;
+import java.util.List;
+import java.util.Optional;
+
+// FIXME: GearInstance is missing Powder Specials...
```

Comment:
How so?

Reply:
The powder slots we can see only tells you about the type of powder added (and how many), not their tier. If you have I think it is two tier 4 or above, you get the Powder Special. This is listed in the tooltip. We should be able to parse that. This is data that belongs in a GearInstance, since it is for a specific instance. If we knew this, we could (for instance) send it along with the chat encoding (but that would mean having a way to modify the protocol, which we have talked about in other situations).

In any way, this is just a reminder that there is more we could possibly extract, not that we are missing something we did before. I intended to fix this originally, but since it all just grew out of hand I had to leave it out.